packages/babel-preset-env/src/polyfills/corejs2/usage-plugin.js

```diff
@@ -0,0 +1,224 @@
+      if (!path.get("callee.property").matchesPattern("Symbol.iterator")) {
+        return;
+      }
+
+      this.addImport("web.dom.iterable");
+    },
+
+    // Symbol.iterator in arr
+    BinaryExpression(path) {
+      if (path.node.operator !== "in") return;
+      if (!path.get("left").matchesPattern("Symbol.iterator")) return;
+
+      this.addImport("web.dom.iterable");
+    },
+
+    // yield*
+    YieldExpression(path) {
+      if (path.node.delegate) {
+        this.addImport("web.dom.iterable");
+      }
+    },
+
+    // Array.from
+    MemberExpression: {
+      enter(path) {
+        if (!path.isReferenced()) return;
+
+        const { node } = path;
+        const { object, property } = node;
+
+        if (!t.isReferenced(object, node)) return;
+
+        let evaluatedPropType = object.name;
+        let propertyName = property.name;
+        let instanceType;
+
+        if (node.computed) {
+          if (t.isStringLiteral(property)) {
+            propertyName = property.value;
+          } else {
+            const result = path.get("property").evaluate();
+            if (result.confident && result.value) {
+              propertyName = result.value;
+            }
+          }
+        }
+
+        if (path.scope.getBindingIdentifier(object.name)) {
+          const result = path.get("object").evaluate();
+          if (result.value) {
+            instanceType = getType(result.value);
+          } else if (result.deopt && result.deopt.isIdentifier()) {
+            evaluatedPropType = result.deopt.node.name;
+          }
+        }
+
+        if (has(StaticProperties, evaluatedPropType)) {
+          const BuiltInProperties = StaticProperties[evaluatedPropType];
+          if (has(BuiltInProperties, propertyName)) {
+            const StaticPropertyDependencies = BuiltInProperties[propertyName];
+            this.addUnsupported(StaticPropertyDependencies);
+          }
+        }
+
+        if (has(InstanceProperties, propertyName)) {
+          let InstancePropertyDependencies = InstanceProperties[propertyName];
+          if (instanceType) {
+            InstancePropertyDependencies = InstancePropertyDependencies.filter(
+              module => module.includes(instanceType),
+            );
+          }
+          this.addUnsupported(InstancePropertyDependencies);
+        }
+      },
+
+      // Symbol.match
+      exit(path) {
+        if (!path.isReferenced()) return;
+
+        const { name } = path.node.object;
+
+        if (!has(BuiltIns, name)) return;
+        if (path.scope.getBindingIdentifier(name)) return;
+
+        const BuiltInDependencies = BuiltIns[name];
+        this.addUnsupported(BuiltInDependencies);
+      },
+    },
+
+    // var { repeat, startsWith } = String
+    VariableDeclarator(path) {
+      if (!path.isReferenced()) return;
+
+      const { node } = path;
+      const { id, init } = node;
+
+      if (!t.isObjectPattern(id)) return;
+      if (!t.isReferenced(init, node)) return;
```

Comment:
This is always true, the `isReferenced` function checks if `node.init === init` in this case.